### DISCUSSION TITLE
Valley Bottom Segmentation 2.0

### DISCUSSION THREAD

**Discussion by lauren-herbine** [Riverscapes/riverscapes-tools#564] [General] [answered]
## Segmentation

Some thoughts on a tiered process with some examples. 

This would be using ChannelArea geometries as our main geometry, and then tying the valley bottom segment to it. This is because the ChannelArea polygons have NHD+ data tied to them. The main examples will be from the Alarka Creek- Little Tennessee basin in North Carolina.

### Process

![tiered_process_w_explanations](https://user-images.githubusercontent.com/73319684/163046802-d859bde7-9893-472b-b1b2-6a330814eafe.png)

#### 1. Break out any `FlowArea` FCODEs and make them their own segment

​	*Areas*

​				53700, 46100, 48400, **31800**, **34300, 34305, 34306, 4600, 46003, 46006, 46007**

#### 2. Break out any `Waterbodies` FCODEs and make them their own segment

* *Waterbodies*

49300, 3900, 39001, 39004, 39005, 39006, 39009, 39010, 39011, 39012, 43600, 43601, 43603, 43604, 43605, 43606, 43607, 43608, 43609, 43610, 43611, 43612, 43613, 43614, 43615, 43618, 43619, 43621, 43623, 43624, 43625, 43626, 46600, 46601, 46602

![Alarka_waterbodiesandflowareas](https://user-images.githubusercontent.com/73319684/160141953-9f15bbb4-914d-4ce1-8caa-6da33735c0f8.png)

#### 3. Segment out Perennial

46006

#### 4. Intermittent

46003

#### 5. Ephemeral

46007

These can be either `Flowlines` or `Flowareas`

#### 6. Use LevelPath within VBET to segment

![image](https://user-images.githubusercontent.com/73319684/163050282-17a0ceb3-1900-43ec-be6b-6989acdfef44.png)

#### 7. Segment at Level IV EcoRegion transitions

![Alarka_byecoregions](https://user-images.githubusercontent.com/73319684/160142025-5f70a853-b8e2-44e9-806a-74395658ef6d.png)

#### Tiered segmentation depending on drainage area/stream order for segments still longer than 5 km

* Still need to figure this one out

**Comment by lauren-herbine** [comment]
### A history/catalogue of our discussions and issues relating to this:
[How to create a valley centre line?](https://github.com/Riverscapes/ConfinementTool/issues/39)
Decide and Implement new Centerline Strategy #382 
Segmentation of Valley Bottom Suggestions & Observatons #414 
Key to New Riverscape Network - Topology & "hydroenforcement" #419

**Comment by lauren-herbine** [comment]
@joewheaton @philipbaileynar 
Has there been further discussion on this? Or is this still the line of thinking?

**Comment by lauren-herbine** [comment]
From call with @philipbaileynar , @KellyMWhitehead , @shelbysawyer , @joewheaton , @lauren-herbine on 4/21/22:

## Updated 4/21

### LevelPaths and Centerlines

* We start at LevelPaths
* Centerline production:
  * produce VBET evidence for each levelpath (LP)
    * this translates into HAND for each trib
    * and a valley bottom (VB) extent for each flowline
  * run Euclidean distance from edge of raster, which produces an exaggerated ridge
  * multiply by -1 to get a valley
  * run accumulation/cost surface analysis
    * stays in middle of valley to accumulate least amount of cost-> *centerline!*

#### Pros

* can combine LPs for things w/ NHD attributes w/ diffluences/confluences to make one VB
  * side channel LPs will be run with their parent level path

#### Issues

* which level path do we run?
  * how do we make sure we only run the ones we want to run?
* lobes go up into tribs, likely due to using old transform functions
  * might need to weight HAND >>> slope and TWI
* FlowAreas likely don't have LPs (maybe but Lauren will do more research...)
* Waterbodies likely have LPs (maybe but Lauren will do more research...)

### Segmentation

1. LevelPaths
2. Waterbodies
3. Ecoregions
4. Stream size (some sort of 1, 2, 3) 
5. Perennial, intermittent, ephemeral

#### Issues related to segmentation

* reliably finding the top and bottom of these features
* doing the actual split- finding the perpendicular
* ecoregions might produce "orphaned" VBs (fine with Joe, Lauren doesn't think this will be an issue too much. Ecoregions, from how they are formed, follow geology pretty closely. They therefore will *tend* to cut somewhat perpendicularly across riverscapes. Lauren can do some zooming around to confirm)
* Perennial, intermittent, ephemeral- looking upstream and downstream- am I in a checkerboard? Do we really want to split these out?
  * if it is making too small of a break, ignore it

##### some visuals from the call:

![call1](https://user-images.githubusercontent.com/73319684/164554029-bc556884-fa97-4387-8ee6-b6af6843446e.PNG)

![call2_waterbodies](https://user-images.githubusercontent.com/73319684/164554103-22fe1589-628f-4630-b7e3-7dbda2b975e5.PNG)

**Comment by shelbysawyer** [comment, marked as answer]
[Here is a gist](https://gist.github.com/shelbysawyer/6835fc0a4ddb05632f3d6f676005fe35) that I wrote up on what this rule-based approach looks like. @lauren-herbine Think we can mark this discussion as done for now and open it back up if we come back to it?

**Comment by lauren-herbine** [comment]
Yes, I agree. Closing
